nav2_dwb_controller/dwb_core/src/publisher.cpp

```diff
@@ -159,62 +159,66 @@ void
 DWBPublisher::publishTrajectories(const dwb_msgs::msg::LocalPlanEvaluation & results)
 {
   if (!publish_trajectories_) {return;}
-  visualization_msgs::msg::MarkerArray ma;
-  visualization_msgs::msg::Marker m;
 
   if (results.twists.size() == 0) {return;}
 
-  geometry_msgs::msg::Point pt;
+  if (node_->count_subscribers(marker_pub_->get_topic_name()) > 0) {
```

Comment:
Maybe I wasn't clear here - what I'm asking for if for this change to follow the convention in the file. If you look above now, you have 2 checks for invalid that do not nest the rest of the function. Rather than

```
if (true) {
  ...
  entire function code
  ...
}
```

it has 
```
if (false) {return;}
```

That's what I'm asking for here. 

```
if (node_->count_subscribers(marker_pub_->get_topic_name()) < 1) {return;}
```

so the entire function isn't nested.

Reply:
My bad!